careerfairsystems/arkad-app-2018
# [#77] Add list to house view

## Change description

New house view with map and list of companies in current map. It's possible to click on a company in the list to see were it is on the map.

## How to verify

Click on all the companies in every map and check that they have a location on the map. 

## Issues fixed

Fixes #77
